sql/firefox_kpi_dashboard_v1.sql

```diff
@@ -0,0 +1,231 @@
+CREATE TEMP FUNCTION
+  sum_buckets_with_ci(a ARRAY<INT64>)
```

Comment:
This seems generic enough to be copied to `udf/udf_sum_buckets_with_ci.js.sql`, and should have a `udf_` prefix for consistency:
```suggestion
  udf_sum_buckets_with_ci(a ARRAY<INT64>)
```